### PR TITLE
Sisrp 7842

### PR DESCRIPTION
**Pull request by marcpechaitis**

### CODE REVIEW

**Comment by marcpechaitis** [comment]
@christianv please review and merge.  thank you!

File: src/css/cs/items/_kitchensink.scss [hunk 1]
```diff
@@ -271,7 +271,7 @@
 
   // Custom font-widths, PS does not deliver classes to allow different font-sizes
   @for $i from 10 through 25 {
-    $width: $i * 1px;
+    $width: $i * 5px;
```
Comment (christianvuerings) [inline src/css/cs/items/_kitchensink.scss:274]: @marcpechaitis was this change intended? For font's it should be fine to have 10,11,12px but not for margins / padding

File: src/css/cs/prototypes/_advising.scss [hunk 2]
```diff
@@ -325,8 +325,3 @@
     }
   }
 }
-
-// Hide the global load icon
-.psc_processing {
```
Comment (christianvuerings) [inline src/css/cs/prototypes/_advising.scss:330]: @marcpechaitis lgtm

File: src/css/cs/items/_kitchensink.scss (hunk 1 above)
Comment (marcpechaitis) [inline src/css/cs/items/_kitchensink.scss:274]: No that wasn't intended.  Not sure why I don't have the margin-width showing up in my local any more?..i'm looking into it.

In the meantime I'm changing the font-width back to "1px"

Comment (christianvuerings) [inline src/css/cs/items/_kitchensink.scss:274]: @marcpechaitis definitely ping me with @christianv when you've updated this PR.

**Comment by christianvuerings** [comment]
Follow up at #38
